Tasks/NodeToolV0/task.json

```diff
@@ -66,6 +66,14 @@
             "defaultValue": "false",
             "required": false,
             "helpMarkDown": "Installs the x86 version of Node regardless of the CPU architecture of the agent."
+        },
+        {
+          "name": "nodejsMirror",
+          "type": "string",
+          "label": "Set an alternative source for Node.js binaries",
```

Comment:
My suggestion is to move this option to advanced group, and change the description. Because when we have default value 'an alternative source' seems a bit stranger.

Reply:
> "move this option to advanced group"

What does "advanced group" mean? I have searched the codebase and cannot find what "advanced group" might refer to.

I have reworded the text. If it is not to your satisfaction then please may I ask that you provide an alternative suggestion.

Reply:
You can refer MSBuildV1 task.json, there might be what I prefer. 

Regarding label my suggestion is "Set source for Node.js binaries" and you can move current label to helpMarkdown to give more detailed information about this input

Reply:
> "... refer MSBuildV1 task.json,"

[(commit)](https://github.com/microsoft/azure-pipelines-tasks/pull/17500/commits/b46553a65388b4280a1366aeb61bc5f25b5240ea) I have created a new Advanced Group and marked the `nodejsMirror` input as belonging to that `"advanced"` group. 

> "my suggestion is 'Set source for Node.js binaries'"

[(commit)](https://github.com/microsoft/azure-pipelines-tasks/pull/17500/commits/35f00f952ddf1d8c3e8f406d3635bd37d194a3aa) I have reworded the value of the `label` text to exactly what you have asked.

> "can move current label to helpMarkdown"

Sorry @ismayilov-ismayil, I do not understand what you are asking me to do. Yes, I have a `label` value and I have a `helpMarkdown` for the new input: `nodejsMirror`, and I have set the `label` value to the text you wanted. Are you asking me to remove `label` completely and include only a `helpMarkdown` field with the text you have suggested? Please clarify.

Reply:
@trippingtarballs sorry if I couldn't explain clearly. 
You are already done what I suggested. Thanks